src/db/models/User.js

```diff
@@ -44,11 +44,22 @@ export default (sequelize, DataTypes) => {
             args: 8,
             msg: 'Password must be at least 8 characters long'
           },
-          isAlphanumeric: {
-            msg: 'Password should be alphanumeric e.g. abc123'
+          isAlphanumeric(value) {
```

Comment:
Please, can you throw more light on what's going on here?

Reply:
I am hashing the password before updating the password field, so due to the validation going on the database, the regex function was made to help me save the hashed password.